### PR TITLE
Pass the correct attribute to handle_stylesheet_url.

### PR DESCRIPTION
**Pull request by Ms2ger**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13461)

### CODE REVIEW

**Comment by Ms2ger** [comment]
Found this while looking into https://github.com/servo/servo/issues/13462

File: tests/wpt/web-platform-tests/html/semantics/document-metadata/the-link-element/stylesheet-media.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Test</title>
+<link rel=match href=stylesheet-media-ref.html>
+<style>
```
Comment (Manishearth) [inline tests/wpt/web-platform-tests/html/semantics/document-metadata/the-link-element/stylesheet-media.html]: Please add a description to explain what behavior we're testing here.

**Review by Manishearth — APPROVED** [review]
r=me with nit

**Pull request merged**